virttest/env_process.py

```diff
@@ -1044,6 +1044,14 @@ def preprocess(test, params, env):
         iscsidev = qemu_storage.Iscsidev(params, base_dir, "iscsi")
         params["image_name"] = iscsidev.setup()
         params["image_raw_device"] = "yes"
+        for image_name in params.objects("images"):
+            lun_tag = "lun_%s" % image_name
+            if params.get(lun_tag):
+                image_params = params.object_params(image_name)
+                lun = image_params.get("lun", "0")
+                name_tag = "image_name_%s" % image_name
+                params[name_tag] = iscsidev.iscsidevice.get_devices()[lun]
+                params["image_raw_device_%s" % image_name] = "yes"
```

Comment:
I just noticed that image_raw_device is set to raw in guest-hw.cfg, so here we may remove this line

Reply:
Although we define image_raw_device = yes in guest-hw.cfg, it doesn't take effect when rebase. Dut to the code limitation,  we define image_raw_device_$snapshot = no, in qemu_storage.rebase, we get image_raw_device = no for base image, that's not reasonable.

If we define params["image_raw_device_%s" % image_name] = "yes" in env_process, we can get image_raw_device = yes for base image when we do rebasing.

Reply:
Sure, I got that pain-point, then it's OK for me for the update